renderer/containers/action-bar.js

```diff
@@ -91,8 +91,14 @@ export default class ActionBarContainer extends Container {
 
   toggleAdvanced = () => {
     if (!this.cropperContainer.state.isFullscreen) {
-      const {advanced} = this.state;
+      const {advanced, screenWidth, screenHeight} = this.state;
       this.updateSettings({advanced: !advanced});
+      if (!advanced) {
+        this.cropperContainer.setSize({
+          width: screenWidth * 0.2,
```

Comment:
Maybe this should use the same logic as the other stuff to get 4:3 aspect?

Reply:
added height calculation depending from width